From 6756cfd50be69d2e22adf05620fa7ffc7b0c1ec0 Mon Sep 17 00:00:00 2001
From: Sandro Tosi <morph@debian.org>
Date: Wed, 17 Jan 2018 20:20:58 -0500
Subject: disable intersphinx

---
 doc/conf.py | 4 ----
 1 file changed, 4 deletions(-)

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -93,10 +93,6 @@ autodoc_docstring_signature = True
 autodoc_default_flags = ['members', 'undoc-members']
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://docs.scipy.org/doc/numpy/', None),
-    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
-    'pandas': ('https://pandas.pydata.org/pandas-docs/stable', None)
 }
 
 explicit_order_folders = [
