From e3927a2475cbebdd9925d1b61e6d9412321b9e18 Mon Sep 17 00:00:00 2001
From: Sandro Tosi <morph@debian.org>
Date: Wed, 17 Jan 2018 22:34:48 -0500
Subject: remove numpy scipy from reference_url

---
 doc/conf.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/doc/conf.py b/doc/conf.py
index 17b9c5c1..f400223d 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -124,8 +124,6 @@ sphinx_gallery_conf = {
     'doc_module': ('matplotlib', 'mpl_toolkits'),
     'reference_url': {
         'matplotlib': None,
-        'numpy': 'https://docs.scipy.org/doc/numpy',
-        'scipy': 'https://docs.scipy.org/doc/scipy/reference',
     },
     'backreferences_dir': 'api/_as_gen',
     'subsection_order': ExplicitOrder(explicit_order_folders)
