From: Sandro Tosi <morph@debian.org>
Date: Tue, 10 Apr 2018 20:33:24 -0400
Subject: remove distutils.sysconfig

---
 lib/matplotlib/__init__.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/matplotlib/__init__.py b/lib/matplotlib/__init__.py
index 563b0a8..528f2ca 100644
--- a/lib/matplotlib/__init__.py
+++ b/lib/matplotlib/__init__.py
@@ -107,7 +107,6 @@ import atexit
 from collections import MutableMapping
 import contextlib
 import distutils.version
-import distutils.sysconfig
 import functools
 import io
 import inspect
