Description: Fixes path for build on Debian
Forwarded: no
Author: Sandro Tosi <morph@debian.org>

diff -pruN matplotlib-0.98.3.orig/setupext.py matplotlib-0.98.3/setupext.py
--- matplotlib-0.98.3.orig/setupext.py	2008-10-08 23:30:35.000000000 +0000
+++ matplotlib-0.98.3/setupext.py	2008-10-08 23:31:19.000000000 +0000
@@ -47,8 +47,12 @@
 
 basedir = {
     'win32'  : ['win32_static',],
-    'linux2' : ['/usr/local', '/usr'],
-    'linux'  : ['/usr/local', '/usr',],
+    'linux2-alpha' : ['/usr'],
+    'linux2-hppa' : ['/usr'],
+    'linux2-mips' : ['/usr'],
+    'linux2-sparc' : ['/usr'],
+    'linux2' : ['/usr'],
+    'linux'  : ['/usr',],
     'cygwin' : ['/usr/local', '/usr',],
     '_darwin' : ['/sw/lib/freetype2', '/sw/lib/freetype219', '/usr/local',
                 '/usr', '/sw'],
@@ -63,8 +67,10 @@
     'freebsd5' : ['/usr/local', '/usr'],
     'freebsd6' : ['/usr/local', '/usr'],
     'sunos5' : [os.getenv('MPLIB_BASE') or '/usr/local',],
-    'gnukfreebsd5' : ['/usr/local', '/usr'],
-    'gnukfreebsd6' : ['/usr/local', '/usr'],
+    'gnukfreebsd5' : ['/usr'],
+    'gnukfreebsd6' : ['/usr'],
+    'gnukfreebsd7' : ['/usr'],
+    'gnukfreebsd8' : ['/usr'],
     'aix5' : ['/usr/local'],
 }
 
