Description: Fixes path for build on Debian
Forwarded: no
Author: Sandro Tosi <morph@debian.org>
Last-Update: 2010-02-06

--- matplotlib-0.99.1.2.orig/setupext.py
+++ matplotlib-0.99.1.2/setupext.py
@@ -47,8 +47,12 @@ import subprocess
 
 basedir = {
     'win32'  : ['win32_static',],
-    'linux2' : ['/usr/local', '/usr'],
-    'linux'  : ['/usr/local', '/usr',],
+    'linux2-alpha' : ['/usr'],
+    'linux2-hppa' : ['/usr'],
+    'linux2-mips' : ['/usr'],
+    'linux2-sparc' : ['/usr'],
+    'linux2' : ['/usr'],
+    'linux'  : ['/usr',],
     'cygwin' : ['/usr/local', '/usr',],
     '_darwin' : ['/sw/lib/freetype2', '/sw/lib/freetype219', '/usr/local',
                 '/usr', '/sw'],
@@ -63,9 +67,12 @@ basedir = {
     'freebsd5' : ['/usr/local', '/usr'],
     'freebsd6' : ['/usr/local', '/usr'],
     'sunos5' : [os.getenv('MPLIB_BASE') or '/usr/local',],
-    'gnukfreebsd5' : ['/usr/local', '/usr'],
-    'gnukfreebsd6' : ['/usr/local', '/usr'],
+    'gnukfreebsd5' : ['/usr'],
+    'gnukfreebsd6' : ['/usr'],
+    'gnukfreebsd7' : ['/usr'],
+    'gnukfreebsd8' : ['/usr'],
     'aix5' : ['/usr/local'],
+    'gnu0' : ['/usr'],
 }
 
 import sys, os, stat
