From 1a182a98f76ce8c624bbaf3882e40d799e867338 Mon Sep 17 00:00:00 2001
From: Jens Hedegaard Nielsen <jens.nielsen@ucl.ac.uk>
Date: Wed, 29 Oct 2014 19:32:42 +0000
Subject: [PATCH] Use map_async and get to avoid a deadlock

---
 setupext.py | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

--- a/setupext.py
+++ b/setupext.py
@@ -1843,7 +1843,8 @@ class BackendGtk3Cairo(OptionalBackendPa
             p = multiprocessing.Pool()
         except:
             return "unknown (can not use multiprocessing to determine)"
-        success, msg = p.map(backend_gtk3cairo_internal_check, [0])[0]
+        res = p.map_async(backend_gtk3cairo_internal_check, [0])
+        success, msg = res.get(timeout=5)[0]
         p.close()
         p.join()
         if success:
@@ -1979,7 +1980,8 @@ class BackendQtBase(OptionalBackendPacka
         else:
             # Multiprocessing OK
             try:
-                msg = p.map(self.callback, [self])[0]
+                res = p.map_async(self.callback, [self])
+                msg = res.get(timeout=5)[0]
             except:
                 # If we hit an error on multiprocessing raise it
                 raise
