Description: don't separate param and its argument with a space

Index: matplotlib-1.0.1/lib/mpl_toolkits/axes_grid1/axes_divider.py
===================================================================
--- matplotlib-1.0.1.orig/lib/mpl_toolkits/axes_grid1/axes_divider.py	2011-01-13 19:25:40.413682190 +0100
+++ matplotlib-1.0.1/lib/mpl_toolkits/axes_grid1/axes_divider.py	2011-01-13 18:59:00.990682216 +0100
@@ -172,12 +172,12 @@
     def locate(self, nx, ny, nx1=None, ny1=None, renderer=None):
         """
 
-        :param nx, nx1: Integers specifying the column-position of the
+        :param nx,nx1: Integers specifying the column-position of the
           cell. When nx1 is None, a single nx-th column is
           specified. Otherwise location of columns spanning between nx
           to nx1 (but excluding nx1-th column) is specified.
 
-        :param ny, ny1: same as nx and nx1, but for row positions.
+        :param ny,ny1: same as nx and nx1, but for row positions.
         """
 
 
@@ -222,12 +222,12 @@
         (:class:`mpl_toolkits.axes_grid.axes_divider.AxesLocator`) for
         specified cell.
 
-        :param nx, nx1: Integers specifying the column-position of the
+        :param nx,nx1: Integers specifying the column-position of the
           cell. When nx1 is None, a single nx-th column is
           specified. Otherwise location of columns spanning between nx
           to nx1 (but excluding nx1-th column) is specified.
 
-        :param ny, ny1: same as nx and nx1, but for row positions.
+        :param ny,ny1: same as nx and nx1, but for row positions.
         """
         return AxesLocator(self, nx, ny, nx1, ny1)
 
@@ -242,12 +242,12 @@
         """
         :param axes_divider: An instance of AxesDivider class.
 
-        :param nx, nx1: Integers specifying the column-position of the
+        :param nx,nx1: Integers specifying the column-position of the
           cell. When nx1 is None, a single nx-th column is
           specified. Otherwise location of columns spanning between nx
           to nx1 (but excluding nx1-th column) is is specified.
 
-        :param ny, ny1: same as nx and nx1, but for row positions.
+        :param ny,ny1: same as nx and nx1, but for row positions.
         """
         self._axes_divider = axes_divider
 
