Index: matplotlib-0.91.2/ttconv/truetype.h
===================================================================
--- matplotlib-0.91.2.orig/ttconv/truetype.h	2008-04-28 20:13:10.173119651 +0200
+++ matplotlib-0.91.2/ttconv/truetype.h	2008-04-28 20:14:38.434149359 +0200
@@ -20,6 +20,10 @@
 ** Last modified 19 April 1995.
 */
 
+/* Debian BTS 478237: FTFS with gcc4.3
+ * Thanks to Soeren Sonnenburg <bugreports@nn7.de> */
+#include <stdio.h>
+
 /* Types used in TrueType font files. */
 #define BYTE unsigned char
 #define USHORT unsigned short int
