Index: matplotlib-0.98.3/src/mplutils.cpp
===================================================================
--- matplotlib-0.98.3.orig/src/mplutils.cpp	2008-11-13 23:32:49.541833541 +0100
+++ matplotlib-0.98.3/src/mplutils.cpp	2008-11-13 23:32:59.197836373 +0100
@@ -1,5 +1,6 @@
 #include <iostream>
 #include <cstdarg>
+#include <cstdio>
 #include "mplutils.h"
 
 void _VERBOSE(const std::string& s) {
