Index: matplotlib-0.98.3/setupext.py
===================================================================
--- matplotlib-0.98.3.orig/setupext.py	2008-08-05 17:43:30.063494149 +0200
+++ matplotlib-0.98.3/setupext.py	2008-08-05 17:47:59.774864127 +0200
@@ -47,8 +47,8 @@
 
 basedir = {
     'win32'  : ['win32_static',],
-    'linux2' : ['/usr/local', '/usr'],
-    'linux'  : ['/usr/local', '/usr',],
+    'linux2' : ['/usr'],
+    'linux'  : ['/usr',],
     'cygwin' : ['/usr/local', '/usr',],
     'darwin' : ['/sw/lib/freetype2', '/sw/lib/freetype219', '/usr/local',
                 '/usr', '/sw', '/usr/X11R6'],
@@ -56,8 +56,10 @@
     'freebsd5' : ['/usr/local', '/usr'],
     'freebsd6' : ['/usr/local', '/usr'],
     'sunos5' : [os.getenv('MPLIB_BASE') or '/usr/local',],
-    'gnukfreebsd5' : ['/usr/local', '/usr'],
-    'gnukfreebsd6' : ['/usr/local', '/usr'],
+    'gnukfreebsd5' : ['/usr'],
+    'gnukfreebsd6' : ['/usr'],
+    'gnukfreebsd7' : ['/usr'],
+    'gnukfreebsd8' : ['/usr'],
     'aix5' : ['/usr/local'],
 }
 
@@ -977,6 +979,10 @@
         tk_inc = os.path.normpath(os.path.join(tk_lib_dir,
                                                        '../../include'))
 
+    if not os.path.exists(tk_inc):
+        tk_inc = os.path.normpath(os.path.join(tk_lib_dir,
+                                               '../../../include/tcl' + tk_ver))
+
     if not os.path.exists(os.path.join(tk_inc, 'tk.h')):
         tk_inc = tcl_inc
 
@@ -994,10 +1000,10 @@
     return tcl_lib, tcl_inc, tk_lib, tk_inc
 
 def hardcoded_tcl_config():
-    tcl_inc = "/usr/local/include"
-    tk_inc = "/usr/local/include"
-    tcl_lib = "/usr/local/lib"
-    tk_lib = "/usr/local/lib"
+    tcl_inc = "/usr/include/tcl8.4"
+    tk_inc = "/usr/include/tcl8.4"
+    tcl_lib = "/usr/lib"
+    tk_lib = "/usr/lib"
     return tcl_lib, tcl_inc, tk_lib, tk_inc
 
 def add_tk_flags(module):
