Index: matplotlib/setupext.py
===================================================================
--- matplotlib.orig/setupext.py	2008-04-22 22:50:11.272065078 +0200
+++ matplotlib/setupext.py	2008-04-22 23:33:33.044331630 +0200
@@ -47,8 +47,8 @@
 
 basedir = {
     'win32'  : ['win32_static',],
-    'linux2' : ['/usr/local', '/usr'],
-    'linux'  : ['/usr/local', '/usr',],
+    'linux2' : ['/usr'],
+    'linux'  : ['/usr',],
     'cygwin' : ['/usr/local', '/usr',],
     'darwin' : ['/sw/lib/freetype2', '/sw/lib/freetype219', '/usr/local',
                 '/usr', '/sw', '/usr/X11R6'],
@@ -56,8 +56,8 @@
     'freebsd5' : ['/usr/local', '/usr'],
     'freebsd6' : ['/usr/local', '/usr'],
     'sunos5' : [os.getenv('MPLIB_BASE') or '/usr/local',],
-    'gnukfreebsd5' : ['/usr/local', '/usr'],
-    'gnukfreebsd6' : ['/usr/local', '/usr'],
+    'gnukfreebsd5' : ['/usr'],
+    'gnukfreebsd6' : ['/usr'],
     'aix5' : ['/usr/local'],
 }
 
@@ -960,6 +960,9 @@
             if not os.path.exists(tk_inc):
                 tk_inc = os.path.normpath(os.path.join(tk_lib_dir, 
                                           '../../include'))
+            if not os.path.exists(tk_inc):
+                tk_inc = os.path.normpath(os.path.join(tk_lib_dir,
+                                          '../../../include/tcl' + tk_ver))
             
             if ((not os.path.exists(os.path.join(tk_inc,'tk.h'))) and
                 os.path.exists(os.path.join(tcl_inc,'tk.h'))):
@@ -977,11 +980,11 @@
 Using default library and include directories for Tcl and Tk because a
 Tk window failed to open.  You may need to define DISPLAY for Tk to work
 so that setup can determine where your libraries are located."""
-            tcl_inc = "/usr/local/include"
-            tk_inc = "/usr/local/include"
-            tcl_lib = "/usr/local/lib"
-            tk_lib = "/usr/local/lib"
-            tk_ver = ""
+            tcl_inc = "/usr/include/tcl8.4"
+            tk_inc = "/usr/include/tcl8.4"
+            tcl_lib = "/usr/lib"
+            tk_lib = "/usr/lib"
+            tk_ver = "8.4"
         # Add final versions of directories and libraries to module lists
         module.include_dirs.extend([tcl_inc, tk_inc])
         module.library_dirs.extend([tcl_lib, tk_lib])
