Index: matplotlib-0.98.3/doc/sphinxext/inheritance_diagram.py
===================================================================
--- matplotlib-0.98.3.orig/doc/sphinxext/inheritance_diagram.py	2008-08-04 20:38:45.826256001 +0200
+++ matplotlib-0.98.3/doc/sphinxext/inheritance_diagram.py	2008-08-04 20:39:01.087125677 +0200
@@ -346,7 +346,7 @@
     parts = node['parts']
 
     graph_hash = get_graph_hash(node)
-    name = "inheritance%s"
+    name = "inheritance%s" % graph_hash
     pdf_path = os.path.join('_static', name + ".pdf")
 
     graph.run_dot(['-Tpdf', '-o%s' % pdf_path],
