Index: matplotlib-0.98.1/MANIFEST.in
===================================================================
--- matplotlib-0.98.1.orig/MANIFEST.in	2008-09-14 11:23:38.000000000 -0500
+++ matplotlib-0.98.1/MANIFEST.in	2008-09-14 11:24:01.000000000 -0500
@@ -15,6 +15,7 @@
 include lib/matplotlib/mpl-data/fonts/afm/*
 recursive-include license LICENSE*
 recursive-include examples *
+recursive-include doc *
 recursive-include src  *.cpp *.c *.h
 recursive-include CXX  *.cxx *.hxx *.c *.h
 recursive-include agg24 *
