diff -Nurp matplotlib-0.98.3/matplotlibrc.template matplotlib-0.98.3.new/matplotlibrc.template
--- matplotlib-0.98.3/matplotlibrc.template	2008-08-03 20:15:22.000000000 +0200
+++ matplotlib-0.98.3.new/matplotlibrc.template	2008-10-14 20:37:17.000000000 +0200
@@ -4,7 +4,7 @@
 # of it on your system in
 # site-packages/matplotlib/mpl-data/matplotlibrc.  If you edit it
 # there, please note that it will be overridden in your next install.
-# If you want to keep a permanent local bopy that will not be
+# If you want to keep a permanent local copy that will not be
 # over-written, place it in HOME/.matplotlib/matplotlibrc (unix/linux
 # like systems) and C:\Documents and Settings\yourname\.matplotlib
 # (win32 systems).
