From 34636bfe2b9fa9dfa0eb6db6dc662c9b57fe79d6 Mon Sep 17 00:00:00 2001
From: Jody Klymak <jklymak@gmail.com>
Date: Sat, 20 Nov 2021 08:28:56 +0100
Subject: [PATCH 1/2] FIX: don't double transpose xy for add-lines colorbar

---
 lib/matplotlib/colorbar.py | 2 --
 1 file changed, 2 deletions(-)

--- a/lib/matplotlib/colorbar.py
+++ b/lib/matplotlib/colorbar.py
@@ -783,8 +783,6 @@ class Colorbar:
         xy[[2, 3], 1] += fac
         # back to axes units...
         xy = self.ax.transAxes.inverted().transform(inches.transform(xy))
-        if self.orientation == 'horizontal':
-            xy = xy.T
         col.set_clip_path(mpath.Path(xy, closed=True),
                           self.ax.transAxes)
         self.ax.add_collection(col)
